config/config.go

```diff
@@ -581,7 +582,16 @@ func LoadConfigFile(config string) ([]byte, error) {
 	}
 
 	// If it isn't a https scheme, try it as a file
-	return os.ReadFile(config)
+	buffer, err := os.ReadFile(config)
+	if err != nil {
+		return nil, err
+	}
+
+	if !util.IsText(buffer) {
+		return nil, fmt.Errorf("provided config is not a text file")
```

Comment:
Can we do something like this to be clear as to what file we are talking about?

```suggestion
		return nil, fmt.Errorf("provided config is not a TOML file: %s", config)
```